### PR TITLE
feat: Add extra information to README

### PR DESCRIPTION
**Pull request by marharyta**
Self-explanatory, i guess :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,8 +98,23 @@ yarn start
...
+
+- map with markers of places
+- list of places of interest by type and category
+- views and UIs that comply with Helsinki City brand guidelines
+- localization
+- Test experience views
+- accesibility
+- server side rendering
+- visual and programmatic testing
+- UX reserach and multiple use cases
```
Comment (ahie) [inline README.md]: > reserach

nit: typo.

File: README.md [hunk 2]
```diff
@@ -74,8 +98,23 @@ yarn start
...
+For now scope of the project includes:
+
+- map with markers of places
+- list of places of interest by type and category
+- views and UIs that comply with Helsinki City brand guidelines
+- localization
+- Test experience views
+- accesibility
+- server side rendering
+- visual and programmatic testing
```
Comment (ahie) [inline README.md]: > visual

e2e testing is something that should also be added to the new repo.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
+
+## Map
+The map we use is Mapbox:
+- minimal style: `mapbox://styles/strawshield/cjxx7z1sf04rm1dl7bjryf4xf` designed by Olli Kilpi
+
+## Learn More
+The app has been bootstrapped from Create React App.  [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+## The setup
+We have a Google Analytics team, "Ahti Helsinki".
```
Comment (ahie) [inline README.md]: > We have a Google Analytics team, "Ahti Helsinki".

We need to set this up so everyone in the current team has access.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,30 @@
...
+## General information
+
+https://drive.google.com/open?id=1Rghr1WjKDJs0Vmbkmp1yLyF6IxmX5lGn
+
+## Design sources
+Are in Figma https://www.figma.com/file/ZKAH36THbAPVs9gNdmH8N6Dh/Ahti?node-id=0%3A1
+
+## Map
+The map we use is Mapbox:
+- minimal style: `mapbox://styles/strawshield/cjxx7z1sf04rm1dl7bjryf4xf` designed by Olli Kilpi
```
Comment (ahie) [inline README.md]: This style is no longer in use as we do not want to be dependent on Mapbox' paid services. Instead we use tiles provided by City of Helsinki (https://gitlab.com/City-of-Helsinki/ohjelmistokehitys/-/wikis/maps) and styles based on OSM-liberty (https://github.com/maputnik/osm-liberty). The map style is still WIP and needs design.

Comment (marharyta) [reply README.md]: Thanks a for the clarification, @ahie ! Can we remove it for now? The styles are stil a subject for a change in this project :sigh

File: README.md (hunk 2 above)
Comment (marharyta) [reply README.md]: Thanks!

File: README.md (hunk 1 above)
Comment (marharyta) [reply README.md]: oooops

File: README.md (hunk 3 above)
Comment (marharyta) [reply README.md]: I agree!

File: README.md (hunk 4 above)
Comment (ahie) [reply README.md]: Could remove the whole Map section for now, yes.

**Pull request merged**